### ISSUE TITLE
Feature request for ocrd workspace 

### ISSUE THREAD

**Issue by VolkerHartmann** [OCR-D/core#99]
It would be nice to 
- `list-group`: list the USE attribute of all file groups. 
- `list-id`: list the group ID of all files in a file group
- ``: get mimetype of a file referenced by USE and GROUPID or by ID

**Comment by kba** [comment]
The first one (`list-group`) is easy, #102 

The second one (`list-id`): IIUC, the easiest way would be to allow overriding the output field. Then you could do:

```
ocrd workspace -d /path/to/bla find --file-grp OCR-D-IMG --output-field group_id
```

This would also cover the third one, though I don't see why you'd need both the `USE` and `id`, since `id` must be unique within the METS.

```
ocrd workspace -d /path/to/bla find --file-id FILEID_3000 --output-field mimetype
```

**Comment by VolkerHartmann** [comment]
I like the option to define an output field.
But it's not a field it's mostly an attribute, right?
Possible values are: None (file URL), USE , GROUPID, ID, MIMETYPE
It's tricky to implement due to the attributes are defined in different Elements.

**Comment by kba** [comment]
Fixed by #102 #103
